cmd/otelauto/main.go

```diff
@@ -49,10 +68,19 @@ func main() {
 	}
 
 	slog.Info("Starting main node")
-	// TODO: add shutdown hook for graceful stop
 	bp.Run(context.TODO())
 }
 
+func erasePinnedMaps() {
+	for _, m := range fs.PinnedMaps {
+		slog.Debug("cleaning storage used by pinned object", "map", m)
+		err := os.Remove(filepath.Join(fs.PinnedRoot, m))
+		if err != nil {
+			slog.Error("can't remove pinned map "+m, err)
+		}
+	}
+}
+
```

Comment:
Related to a previous comment: we should be careful with this. If there are other `otelauto` instances running in the host, will they have their runtime data erased?